### PR TITLE
Unit tests for all model classes with 100% coverage

### LINKED ISSUE #17: Add Unit tests for model classes

### PR DESCRIPTION
**Pull request by egekaradibak**
Fixes #17

### CODE REVIEW

**Review by marta-matu — CHANGES_REQUESTED** [review]
There seems to be an issue with connecting the tests to maven. After running `mvn test` the output states that zero tests were run.

**Comment by egekaradibak** [comment]
thank you, it is fixed!

File: pom.xml [hunk 1]
```diff
@@ -16,13 +16,22 @@
...
   <dependencies>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>4.11</version>
+      <groupId>org.junit</groupId>
+      <artifactId>junit-bom</artifactId>
+      <version>5.8.2</version>
+      <type>pom</type>
+      <scope>import</scope>
```
Comment (marta-matu) [inline pom.xml:29]: There is a warning from maven related to this line:
```[WARNING] 'dependencies.dependency.scope' for org.junit:junit-bom:pom must be one of [provided, compile, runtime, test, system] but is 'import'.```

It looks like 'import' needs to be changed to one of the listed options.

**Review by marta-matu — CHANGES_REQUESTED** [review]

**Review by marta-matu — APPROVED** [review]
Looks great! All tests are executed and pass 👍

**Pull request merged**